test/hotspot/jtreg/compiler/jsr292/methodHandleExceptions/TestAMEnotNPE.java

```diff
@@ -30,8 +30,11 @@
  *
  * @build p.*
  * @run main/othervm compiler.jsr292.methodHandleExceptions.TestAMEnotNPE
- * @run main/othervm -Xint compiler.jsr292.methodHandleExceptions.TestAMEnotNPE
- * @run main/othervm -Xcomp compiler.jsr292.methodHandleExceptions.TestAMEnotNPE
+ * @run main/othervm -Xint
+ *                   compiler.jsr292.methodHandleExceptions.TestAMEnotNPE
+ * @run main/othervm -Xcomp
+ *                   -XX:CompileCommand=compileonly,*::test
```

Comment:
What time you get if you specify all methods in test's class instead:
`compiler.jsr292.methodHandleExceptions.TestAMEnotNPE::*`

Reply:
It takes just a tiny bit longer. `2.3` seconds for that run, whereas with `-XX:CompileCommand=compileonly,*::test` I get `2` seconds.

Reply:
I suggest then to use my proposal - compile all test's methods. I concern that compiling only one method may not catch some issues when the method is inlined.

Reply:
Actually, I just checked what methids it wants to compile. And the generated `test` methods are these:
```
p.Tdirect::test
p.T::test
p.Treflect::test
```

There is also a method `p.I::m` that seems to have been generated. Does that also need to be compiled?
I also see a `p.C::<init>` and `p.E::<init>.` And some class or interface `p.D`. And `p.F::<init>` and `p.F::m`.

@iwanowww do you think we need to include all of these to be compiled?

Reply:
I think we can have several `compileonly` commands for different methods so you can list them all. Please, verify that.

Reply:
I agree that it's easier to reason about the test when all generated methods are compiled.  I see it put generated classes in packages `p` and `q`. Compiling everything from those packages sounds fine.